rskj-core/src/main/java/co/rsk/peg/utils/BrigeEventLoggerLegacyImpl.java

```diff
@@ -150,6 +147,11 @@ public void logBatchPegoutCreated(BtcTransaction btcTx, List<Keccak256> rskTxHas
         throw new DeprecatedMethodCallException("Calling BrigeEventLoggerLegacyImpl.logBatchPegoutCreated method after RSKIP146 activation");
     }
 
+    @Override
+    public void logPegoutConfirmed(Sha256Hash btcTxHash, long pegoutCreationRskBlockNumber) {
```

Comment:
We should add a test for a call to this method pre RSKIP 146

Reply:
This pr is only about declaring the new event and adding this simple test. More tests for this event will be added in a different story which is about actually emitting this event.

Reply:
That's fine by me as long as you add the details to the corresponding story or create a new one, to make sure we don't forget about this.

It's a simple test anyway, just checking that you get an exception when calling this method pre RSKIP146